tests/server/test_loggers.py

```diff
@@ -98,6 +99,7 @@ def test_data_logging_from_predefined():
         assert log == expected_log
 
 
+@flaky(max_runs=4, min_passes=3)
```

Comment:
how long does this test take? any reason not to do 2:1?

Reply:
It's a quick one, a few ms.
0.75% is more tough to pass then 0.5% ;)